.claude/hooks/workflow-state.sh

```
@@ -12,21 +15,96 @@ _wf_file() {
+    tsconfig.json tsconfig.base.json \
+    biome.json turbo.json drizzle.config.ts lefthook.yml; do
+    [ -f "$root/$f" ] && root_files="$root_files $root/$f"
+  done
+
+  [ -z "$roots" ] && [ -z "$root_files" ] && return 0
+
+  # (b) mtime comparison. -prune excludes heavy dirs; -path prune for nested
+  # worktrees so other branches' edits don't invalidate this branch's gates.
+  # shellcheck disable=SC2086
+  local newer
+  newer=$(find $roots $root_files \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)